src/components/viewer.module.css

```diff
@@ -1,12 +1,15 @@
 .container {
     height: calc(
         100vh - var(--nav-bar-height) - var(--reaction-panel-height) -
-            var(--content-height)
+            var(--content-height) + (var(--border-thickness) * 2)
     );
 }
 
 .container.example {
-    height: calc(100vh - var(--nav-bar-height) - var(--content-height));
+    height: calc(
+        100vh - var(--nav-bar-height) - var(--content-height) +
+            (var(--border-thickness) * 2)
+    );
```

Comment:
in doing this, I noticed there was a 2 px space on the bottom that I wanted to remove (caused by not including 2 borders in height calc)